src/utils/convertPartialDateStringToObj.test.ts

```diff
@@ -0,0 +1,88 @@
+import convertPartialDateStringToObj from './convertPartialDateStringToObj';
+import { DateTime, Settings } from 'luxon';
+
+const currentYear = DateTime.utc().year;
+
+beforeAll(() => {
+    // Mock Date Time to Sat Jun 29 2019 15:48:12 GMT+0100
+    Settings.now = () => 1561819692628;
+});
+
+afterAll(() => {
+    // Restore Date Time Mock
+    Settings.now = () => Date.now();
+});
+
+describe('convertDateStringToObj', () => {
+    const testCases = [{
+        input: '02/2009',
+        output: { day: 1, month: 2, year: 2009 },
```

Comment:
Use `currentYear` instead of 2009? `2009` is also used in the other test file

Reply:
Wouldn't we need to then use it for the input as well? I think it makes no sense to use it when year is given.

Reply:
Okay, just thought maybe be consistent